src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -3418,9 +3418,6 @@ impl Step for TierCheck {
         );
         cargo.arg(builder.src.join("src/doc/rustc/src/platform-support.md"));
         cargo.arg(builder.rustc(self.test_compiler));
-        if builder.is_verbose() {
-            cargo.arg("--verbose");
-        }
```

Comment:
We're already setting verbosity in `Builder::cargo`, which is indirectly called by `prepare_tool_cargo`, so this is not needed.